.github/workflows/release.yml

```diff
@@ -220,8 +220,8 @@ jobs:
         run: |
           tools pkg repo publish release ${{ needs.prepare-workflow.outputs.salt-version }}
 
-  almalinux-8-pkg-download-tests:
-    name: Test Alma Linux 8 Package Downloads
+  almalinux-8-package-pkg-download-tests:
```

Comment:
Why are we naming these jobs `package-pkg` Seems we are duplicating information

Reply:
Ah, good catch, I can switch that to just be `package` or `onedir`